### PR TITLE
Add Body

### PR DESCRIPTION
**Pull request by TheElementGuy**
Please add more to this pull request!

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -21,6 +21,33 @@
...
+							<div class="w3-container w3-theme-d1">
+								<h5>About</h5>
+							</div>
+							<div class="w3-container w3-theme-l2">
+								<h5>Standings</h5>
+								<h6>Group A</h6>
+								<h6>Group B</h6>
+								<h6>Group C</h6>
+								<h6>Group D</h6>
+								<h6>Group E</h6>
```
Comment (HatchlingsCheeseburger) [inline index.html:45]: `where do I put the table?`

Comment (HatchlingsCheeseburger) [reply index.html:45]: `or tables`

File: index.html [hunk 2]
```diff
@@ -21,6 +21,33 @@
...
+				</div>
+				<div class="w3-container w3-theme-l2">
+					<h2>Sports</h2>
+					<div class="w3-container w3-theme-d5">
+						<h3>Soccer</h3>
+						<div class="w3-container w3-theme-d4" id="andezliga">
+							<h4>Andezliga</h4>
+							<div class="w3-container w3-theme-d1">
+								<h5>About</h5>
+							</div>
```
Comment (HatchlingsCheeseburger) [inline index.html:38]: `where do I put what it is about?`

File: index.html [hunk 3]
```diff
@@ -21,6 +21,33 @@
 		<meta name="theme-color" content="#000000">
```
Comment (HatchlingsCheeseburger) [inline index.html:21]: what theme do you have because I `want` what you have.

Comment (TheElementGuy) [reply index.html:21]: uh... some sort of blu i can put it in when we're done with all the code

Comment (HatchlingsCheeseburger) [reply index.html:21]: NO I WANT IT NOW!

File: index.html (hunk 2 above)
Comment (TheElementGuy) [reply index.html:38]: use like a <p> tag or something under `<h5>About</h5>`

File: index.html (hunk 1 above)
Comment (TheElementGuy) [reply index.html:45]: under the header for that table

File: index.html (hunk 3 above)
Comment (HatchlingsCheeseburger) [reply index.html:21]: `'^'`

File: index.html (hunk 2 above)
Comment (HatchlingsCheeseburger) [reply index.html:38]: ok `'v'`

File: index.html (hunk 1 above)
Comment (HatchlingsCheeseburger) [reply index.html:45]: ok `'v'`

File: index.html (hunk 3 above)
Comment (TheElementGuy) [reply index.html:21]: i did it

**Pull request merged**